pkg/postgres/client.go

```diff
@@ -23,6 +23,8 @@ import (
 	_ "github.com/lib/pq"
 )
 
+const DefaultConnectDatabase = "postgres"
```

Comment:
```suggestion
const defaultConnectDatabase = "postgres"
```

Reply:
We need to export the const as it is being imported and used in another package.